### ISSUE TITLE
[Bug] Styles broke when receiving a response

### ISSUE THREAD

**Issue by YanGusik** [stoplightio/elements#2655]
## Context
If you click on the Send Api Request button, then when outputting, we get a response that is broken in style.

## Current Behavior
visually looks bad for all users

## Steps to Reproduce
1. open [url](https://elements-demo.stoplight.io/#/operations/post-todos)
2. click send api request
3. response 
![изображение_2024-08-14_181317025](https://github.com/user-attachments/assets/3834e3aa-aedc-45d5-bff0-19eb0fd378ea)
style is broken

## Environment
- Link used:
    <script src="https://unpkg.com/@stoplight/elements/web-components.min.js"></script>
    <link rel="stylesheet" href="https://unpkg.com/@stoplight/elements/styles.min.css">
- Environment name and version (e.g. Opera LVL 6 (core: 112.0.5197.60)):
- Operating System and version (desktop or mobile): desktop windows 10

**Comment by wsngamerz** [comment]
I’m also experiencing this issue. As a temporary solution, I pinned the version to the previous working version, which seems to have "resolved" this.

```html
<script src="https://unpkg.com/@stoplight/elements@8.3.4/web-components.min.js"></script>
<link rel="stylesheet" href="https://unpkg.com/@stoplight/elements@8.3.4/styles.min.css">
```

**Comment by beshoo** [comment]
Waiting Updates

**Comment by mojtabaahn** [comment]
+1

**Comment by nikhilbhavsar121** [comment]
> I’m also experiencing this issue. As a temporary solution, I pinned the version to the previous working version, which seems to have "resolved" this.
> 
> ```
> <script src="https://unpkg.com/@stoplight/elements@8.3.4/web-components.min.js"></script>
> <link rel="stylesheet" href="https://unpkg.com/@stoplight/elements@8.3.4/styles.min.css">
> ```

this works for me thanks

**Comment by arno-fukuda** [comment]
**Before**
<img width="500" alt="Screenshot 2024-08-22 at 21 16 08" src="https://github.com/user-attachments/assets/223fc52e-15b9-4c8c-842e-c4cf489bb9b8">

**After** (omg, unbearable)
<img width="500" alt="Screenshot 2024-08-22 at 21 16 51" src="https://github.com/user-attachments/assets/6655f309-3b86-4e1c-96bf-3dd9003f5b39">

**Comment by Gystre** [comment]
This is caused by a bump in version for @stoplight-mosaic from 1.53.1 -> 1.53.3

In this pr: https://github.com/stoplightio/elements/pull/2611

**Comment by mnaumanali94** [comment]
Probably will close this too: https://smartbear.atlassian.net/jira/software/c/projects/STOP/issues/STOP-1043?filter=49162

**Comment by ben-smartbear** [comment]
resolved via https://github.com/stoplightio/elements/pull/2680 and released as 8.4.2
